enhancements/csvless-bundles.md

```diff
@@ -24,209 +24,488 @@ status: implementable
+    path: spec.server.web_root
+    capabilities:
+    - 'urn:alm:descriptor:com.tectonic.ui:label'
+- apiVersion: monitoring.kiali.io/v1alpha1
+  kind: MonitoringDashboard
+  description: A configuration file for defining an individual metric dashboard.
+  displayName: Monitoring Dashboard
   descriptors:
-  - description: The name of the CatalogSource that owns this package
-    displayName: Package Manifest
-    path: status.catalogSource
+  - displayName: Title
+    description: "The title of the dashboard."
+    path: title
     capabilities:
-    - urn:alm:descriptor:text
+    - 'urn:alm:descriptor:com.tectonic.ui:label'
 ```
 
-The `apis` element is a special case and is present to enable to organization of [descriptors](https://github.com/operator-framework/operator-lifecycle-manager#descriptors). It will be kept up to date with `provided APIs` inferred by the `opm bundle build` command upon execution.
+_**Note:** `capabilities` is the equivalent of the `x-discriptors` field found in CSVs._
 
-_**Note:** The `descriptors.path` element's value is qualified with the `status` subresource. For comparision, a CSV contains two types of descriptors -- one for spec, the other for status -- both with path elements that don't contain this additional qualifier._
+The `annotations.yaml` file contains the set of labels to be projected onto the container image representation of the bundle. More importantly, it contains the top-level discriminator field used to identify the mediatype of the bundle on-disk: `operators.operatorframework.io.bundle.mediatype: registry+v2`.
 
-_**Note:** `capabilities` is the equivalent of the `x-descriptors` field found in CSVs._
+#### Build a Bundle Image Without a CSV
 
-### User Stories
+As an Operator Author, I want to build an Operator Bundle image from the same set of Kubernetes manifests I would use to deploy an operator manually, without OLM, so that I can avoid writing a CSV.
 
-#### Packaging an Operator Without a CSV
+##### UX
 
-As an Operator Author, Alice wants to package her operator for use with OLM without writing a CSV, so that she can avoid duplicating deployment configuration that can be expressed with standard Kubernetes resources.
+Add a new command to `opm` to initialize a bundle directory, which has:
 
-Alice has a set of Kubernetes manifests describing her operator already: a single CRD, a Service, a Deployment, and some RBAC rules (a fairly straightforward operator deployment). By providing a minimal amount of additional metadata in a separate file, and without duplicating information present in the existing manifests, Alice can generate an Operator Bundle using the same commands and tooling used were a CSV present.
+- a positional parameter -- `src` -- representing the directory containing source manifests
+- an option for specifying the output directory -- `-o, --out` -- which defaults to the current directory
```

Comment:
Should this be a mandatory positional parameter?

Reply:
Can we remove the default?

I am picking up a `git init` flavor from defaulting the target to `.`, so I checked what happens when you run `git init` in an existing repository:

>      Running git init in an existing repository is safe. It will not
>      overwrite things that are already there. The primary reason for
>      rerunning git init is to pick up newly added templates (or to move the
>      repository to another place if --separate-git-dir is given).

and I think the decision not to avoid destructive behavior is key there.

Reply:
I added that restriction and actually changed things to be more in line with `git init`. How does it look now?